pkgs/by-name/bo/bolt-launcher/package.nix

```
@@ -0,0 +1,78 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  xorg,
+  libarchive,
+  luajit_2_1,
+  libcef,
+  zlib,
+  libGL,
+  autoPatchelfHook,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bolt-launcher";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "AdamCake";
+    repo = "bolt";
+    rev = "${version}";
```

```suggestion
    rev = version;
```

or if you choose the `finalAttrs` route

```suggestion
    rev = finalAttrs.version;
```